doc/classes/PhysicsServer3D.xml

```diff
@@ -582,10 +582,12 @@
 			<param index="1" name="callable" type="Callable" />
 			<param index="2" name="userdata" type="Variant" default="null" />
 			<description>
-				Sets the function used to calculate physics for an object, if that object allows it (see [method body_set_omit_force_integration]).
-				The force integration function takes 2 arguments:
-				[code]state:[/code] [PhysicsDirectBodyState3D] used to retrieve and modify the body's state.
-				[code]userdata:[/code] Optional user data, if it was passed when calling [code]body_set_force_integration_callback[/code].
+				Sets the body's force integration callback function, which can be used e.g. to update the body's linear and angular velocity (for example, based on contact with other bodies).
+				If [param userdata] is not [code]null[/code], the function [param callable] must take the following two parameters:
+				1. a [PhysicsDirectBodyState3D] [code]state[/code]: used to retrieve and modify the body's state,
+				2. a [Variant] [code]userdata[/code]: the user data [param userdata].
+				If [param userdata] is [code]null[/code], then [param callable] must take only the [code]state[/code] parameter.
```

Comment:
So that's what that was. I assumed that the callback was always required to take the userdata as well. Then I should probably fix this for Godot Jolt.

Reply:
It's a slightly odd API, but I just documented how it works in Godot Physics.